### ISSUE TITLE
Dashboard Token Listing: KPTx 

### ISSUE THREAD

**Issue by SFServer** [superfluid-finance/assets#146]
Discord Handle of Submitter: wizgotD #8866

Token Symbol: KPTx

Network: Polygon

Token Address: 0x9556B40E816e2890FdAFfb2fFd47F6493F240781

Network Explorer Link: https://polygonscan.com/address/0x9556b40e816e2890fdaffb2ffd47f6493f240781

Is a Super Token? true

Address of Underlying Token:  0x7D369731E3d7F86417Aa86ef4Be26e309080BD2F

SVG Logo Link: https://drive.google.com/file/d/1Cqfk1YmeliHtX8U3HTQJMO-Sde9efw8H/view?usp=share_link

Coingecko or Hardcoded Price?: Hardcoded

Coingecko ID: 

Harcoded Price (if Coingecko ID is unavailable): 0.0008

Token HEX Color: #2596be

Info on Listing: We are planning to use our token and use the vesting module with our Gnosis Safe DAO module. This would allow our members to see the token directly in dashboard.

Please close only after token has been added to resolver!

**Comment by shaddamciv** [comment]
Hardcoded Price (if Coingecko ID is unavailable): 0.0008 I think this can be 0, I just took the value from dodo.

**Comment by vmichalik** [comment]
listed in resolver, assets pending

**Comment by d10r** [comment]
> Are the assets pending something the team at KP can provide?

yes, you could probably speed up the process by providing a PR for this repo, see previous PRs for how to do it.
